### PR TITLE
refactor user controller with redirect method and user service add so…

### LINKED ISSUE #31: 018-refactor-user-controller
Refactor user controller so that the redirect is made with a function call.

### PR DESCRIPTION
**Pull request by ApostolaT**
…me new-lines

### CODE REVIEW

File: src/Controllers/UserController.php [hunk 1]
```diff
@@ -69,11 +67,7 @@ public function listAll(RequestInterface $request)
     public function goToAddUser(RequestInterface $request)
```
Comment (Raresmldvn) [inline src/Controllers/UserController.php]: return type

File: src/Controllers/UserController.php [hunk 2]
```diff
@@ -89,21 +83,13 @@ public function goToAddUser(RequestInterface $request)
     public function addUser(RequestInterface $request)
```
Comment (Raresmldvn) [inline src/Controllers/UserController.php]: return type

File: src/Controllers/UserController.php [hunk 3]
```diff
@@ -116,21 +102,13 @@ public function addUser(RequestInterface $request)
     public function deleteUser(RequestInterface $request)
```
Comment (Raresmldvn) [inline src/Controllers/UserController.php]: return type

Comment (Raresmldvn) [reply src/Controllers/UserController.php]: and so on for the rest of the methods

File: src/Controllers/UserController.php [hunk 4]
```diff
@@ -86,24 +80,16 @@ public function goToAddUser(RequestInterface $request)
...
 
         $message = ($this->userService->addNewUser($request)) ?
             "Success." : "User addition failed!";
         $this->session->set('message', $message);
-        $response = new Response(Stream::createFromString(' '), []);
-        $response = $response->withStatus(301);
-        $response = $response->withHeader('Location', '/user');
-
-        return $response;
+        return $this->getRedirectPage("/user");
```
Comment (calinpristavu) [inline src/Controllers/UserController.php:95]: blank line before return

**Review by calinpristavu — CHANGES_REQUESTED** [review]

**Pull request merged**